bin/travis/setup.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# pwd is the git repo.
+set -e
+
+echo "Making settings_local.py"
+cat > fjord/settings/local.py <<SETTINGS
+from fjord.settings.base import *
+ROOT_URLCONF = '%s.urls' % PROJECT_MODULE
+LOG_LEVEL = logging.ERROR
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': 'fjord',
+        'USER': 'travis',
+        'HOST': 'localhost',
+        'OPTIONS': {
+            'init_command': 'SET storage_engine=InnoDB',
+            'charset' : 'utf8',
+            'use_unicode' : True,
+        },
+        'TEST_CHARSET': 'utf8',
+        'TEST_COLLATION': 'utf8_general_ci',
+    },
+}
+CELERY_ALWAYS_EAGER = True
+CACHE_BACKEND = 'caching.backends.locmem://'
+ES_INDEX_PREFIX = 'input'
+ES_URLS = ['http://localhost:9200']
+SECRET_KEY = 'TravisShhhhhhhhh'
+DEBUG = True
+SETTINGS
```

Comment:
Instead of doing this, we should have a `fjord/fjord/settings/travis.py` file with the settings in it then set `DJANGO_SETTINGS_MODULE=fjord.settings.travis` in the environment.